fv/wireguard_test.go

```diff
@@ -820,6 +824,37 @@ var _ = infrastructure.DatastoreDescribe("WireGuard-Supported 3 node cluster", [
 				}, "10s", "100ms").Should(BeNumerically(">", 0))
 			}
 		})
+
+		It("WireGuard should be used for host to workload connections on WireGuard enabled nodes", func() {
+			cc.ExpectSome(felixes[0], wls[1])
```

Comment:
`cc.ExpectSome(felixes[1], wls[0])`?

Reply:
No; as that should be exactly symmetric, I thought the encrypted packets testing would be more precise if it was done with just one attempted connection.